status-client-spec.md

```diff
@@ -109,10 +125,43 @@ v5](https://github.com/ethereum/devp2p/blob/master/discv5/discv5.md) and
 [modifications](https://github.com/status-im/rendezvous#differences-with-original-rendezvous)).
 Additionally, some static nodes MAY also be used.
 
-This part of the system is currently underspecified and requires further detail.
+A Status client MUST use at least one discovery method or use static nodes
+to communicate with other clients.
+
+Discovery V5 uses bootstrap nodes to discover other peers. Bootstrap nodes MUST support
+Discovery V5 protocol as well in order to provide peers. It is kademlia-based discovery mechanism
+and it might consume significant (at least on mobile) amount of network traffic to operate.
+
+In order to take advantage from simpler and more mobile-friendly peers discovery mechanism,
+i.e. Rendezvous protocol, one MUST provide a list of Rendezvous nodes which speak
```

Comment:
What is "one"? here? Is this as a bootstrap node?

If a bootstrap node doesn't, what happens? (Should this be a SHOULD?)

Reply:
"one" is a person operating a client in this case.

Rendezvous nodes are sort of bootstrap nodes but speaking the Rendezvous protocol. If someone wants to take advantage of Rendezvous, a list of Rendezvous nodes MUST be provided. The protocol itself does not need to be used though.

Reply:
This should probably be clarified in the spec itself